### PR TITLE
 NMS-13888: Update SNMP Trap docs and stop escaping '=' characters

### PR DESCRIPTION
**Pull request by amay94**
### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13888

### CODE REVIEW

**Comment by amay94** [comment]
Relevant: https://issues.opennms.org/browse/NMS-14102

**Review by cgorantla — COMMENTED** [review]
Think this will not fix existing params. May be we can properly format with the Rest API  ?

**Comment by cgorantla** [comment]
Can we re-target this to `foundation-2021`?

File: docs/modules/operation/pages/events/sources/snmp-traps.adoc [hunk 1]
```diff
@@ -9,6 +9,12 @@ IMPORTANT: Disabling the `Trapd` service daemon will render {page-component-titl
 
 Event definitions are included with {page-component-title} for traps from many vendors' equipment.
 
+== Trap Value Representation
+
+When octet strings are translated into event parameters, OpenNMS first attempts to treat them as character encodings.
+If all bytes in the string are valid UTF-8 or ISO-8859-1 characters, the string is stored as these characters.
+If this is not possible, the value is encoded as a base-64 string.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/events/sources/snmp-traps.adoc:16]:
```suggestion
If this is not possible, the value is encoded as a Base64 string.
```

**Comment by amay94** [comment]
Yeah, I can do that. Should I see if it can easily port back further? I don't think any of this has changed in years

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
One minor change, otherwise docs section looks good.

**Review by Bonrob2 — DISMISSED** [review]
Docs LGTM!

**Comment by cgorantla** [comment]
> Yeah, I can do that. Should I see if it can easily port back further? I don't think any of this has changed in years

Yes,  you can use onto `git rebase --onto foundation-2021 origin/develop`

**Review by cgorantla — APPROVED** [review]

**Pull request merged**